### DISCUSSION TITLE
OCR filling /tmp

### DISCUSSION THREAD

**Discussion by sambrothers** [dadoonet/fscrawler#1231] [Q&A] [answered]
We're using fsCrawler  v2.7 and when we are OCR'ing, the /tmp folder gets filled up with ~74GB of data and then crashes my machine cause / gets full.  Any suggestions?

**Comment by dadoonet** [comment, marked as answer]
Could you share a list of typical files there?

**Reply by cadm-frank** [reply]
We've seen this as well, our workaround is to just run a cronjob every 15min:
```
find $(find /tmp -maxdepth 1 -type d -name 'systemd-private-*-fscrawler.service-*') \( -name 'apache-tika-*.tmp-*' -o -name 'MediaDataBox*' \) -type f -mmin +15 ! -exec fuser -s {} \; -delete
```
I believe it's related to this:
https://github.com/sannies/mp4parser/issues/315
https://github.com/sannies/mp4parser/pull/397

**Reply by dadoonet** [reply]
w00t! That's a great find.

I think this would worth a documentation update. @dfbm would you like to contribute something for this?

**Reply by cadm-frank** [reply]
I'm not sure where in the documentation that should show up?
From my point of view it's a known issue caused by an included library.

**Reply by dadoonet** [reply]
I agree that this should be fixed in the mp4parser project. In the meantime, we could add your script which cleans up the tmp files to the documentation? Like in this page: https://fscrawler.readthedocs.io/en/fscrawler-2.7/user/tips.html?

**Reply by cadm-frank** [reply]
https://github.com/dadoonet/fscrawler/pull/1256
